client/src/app/managing-body/managing-body.component.ts

```diff
@@ -1,18 +1,26 @@
-import { Component, Input } from '@angular/core';
-import { ROUTER_DIRECTIVES } from '@angular/router-deprecated';
+import {Component, Input} from "@angular/core";
```

Comment:
Revert all the import lines to single quotes and add a space inside the curly brackets (like it was).